src/App/StringRole.java

```diff
@@ -0,0 +1,29 @@
+package App;
+
+public class StringRole {
+
+    public String[] SortRole(String[] text, String[] roles) {
```

Comment:
the same

Reply:
simplify this method.
you don't need print only role 
try create implementation without string array